docs/understand/file_reorg.md

```diff
@@ -0,0 +1,120 @@
+# ROCm File Reorganization White Paper
+                                   
+## Introduction
+
+    ROCm packages have adopted the Linux foundation file system hierarchy standard to ensure ROCm components follow open source conventions for Linux-based distributions. Following is the ROCm proposed file structure. 
+    
+    <!-- /opt/rocm-<ver>

```

Comment:
This might be better as a fenced code block (``` ```).

Reply:
If these are general guidelines, we must update our Markdown style guide to include this recommendation. Please confirm.

Reply:
Also, does this flavor of Markdown support syntax highlighting?

Reply:
@saadrahim Is there a recommended code font from the BU? I can check what we used for UIF v1.1 documentation deliverables. Here's the link to the content I created for the GPU content in UIF v1.1:

https://github.com/amd/UIF/tree/main/docs

Reply:
@saadrahim We should consider formatting consistency between rst and markdown files. If not, the content will display differently on the documentation portal.